helm-service/chart/templates/_helpers.tpl

```diff
@@ -61,3 +61,13 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+# preStop hook for helm-service deployment
```

Comment:
pls add the docs comments in the same format as for the other helper functions :)
```suggestion
{{/*
preStop hook for helm-service deployment
*/}}
```